.ci_support/linux_64_.yaml

```diff
@@ -3,6 +3,6 @@ cdt_name:
 channel_sources:
 - conda-forge
 channel_targets:
-- conda-forge main
+- conda-forge openff-interchange_rc
```

Comment:
(not blocking) I've never put one feedstock's packages on another package's custom label, but I don't see anything inherently wrong with it. Just noting this in case it was unintentional.